composer.json

```diff
@@ -19,13 +18,16 @@
         "symfony/framework-bundle": "~2.3",
         "symfony/assetic-bundle": "~2.1",
         "friendsofsymfony/rest-bundle": "~1.0",
-        "jms/serializer-bundle": "~0.12",
         "midgard/createphp": "~1.1"
     },
     "require-dev": {
+        "symfony/serializer": "~2.3",
+        "jms/serializer-bundle": "~0.12|~1.0",
         "symfony-cmf/testing": "~1.2,>=1.2.6"
     },
     "suggest": {
+        "jms/serializer-bundle": "Add support for advanced serialization capabilities, recommended, requires ~0.12|~1.0",
+        "symfony/serializer": "Add support for basic serialization capabilities and xml decoding, requires ~2.3",
```

Comment:
i think we should mention FOSRestBundle here, as we do not interact with the serializer directly.